skfp/applicability_domain/standard_deviation.py

```diff
@@ -0,0 +1,137 @@
+from numbers import Real
+
+import numpy as np
+from sklearn.utils._param_validation import Interval
+from sklearn.utils.validation import validate_data
+
+from skfp.bases.base_ad_checker import BaseADChecker
+
+
+class StdEnsembleADChecker(BaseADChecker):
```

Comment:
`StandardDeviationADChecker` - since standard deviation has to come from an ensemble, it's kind of obvious